src/figuresClasses.ts

```diff
@@ -2,10 +1,88 @@
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = 'triangle';
+
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error('error message');
+    }
+
+    if (a >= b + c || b >= a + c || c >= a + b) {
+      throw new Error('error message');
+    }
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return parseFloat(area.toFixed(2));
+  }
+}
+
+export class Circle implements Figure {
+  shape: 'circle';
 
-export class Triangle implements Figure {}
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public radius: number,
+  ) {
+    this.shape = 'circle';
+
+    if (this.radius <= 0) {
+      throw new Error('error message');
```

Comment:
The error message thrown here is not descriptive. It should explain what went wrong, such as 'Circle radius must be greater than zero.'

Reply:
Consider adding descriptive error message